locales/gl-ES.json

```diff
@@ -208,7 +220,7 @@
         "compose": "Escribir",
         "favourite": "Favorita",
         "title": "Accións",
-        "zen_mode": "Modo Cen"
+        "search": "Buscar"
```

Comment:
`search` should be before `title` here, would you update it?

Reply:
yes, thank you for you help. I've fixed it and made a new PR.